web/app/fr/gospeak/web/pages/partials/container/main.scala.html

```diff
@@ -7,6 +7,7 @@
         <link rel="stylesheet" href="@routes.Assets.versioned("lib/bootstrap/dist/css/bootstrap.min.css")">
         <link rel="stylesheet" href="@routes.Assets.versioned("lib/select2/dist/css/select2.min.css")">
         <link rel="stylesheet" href="@routes.Assets.versioned("lib/select2-bootstrap-theme/dist/select2-bootstrap.min.css")">
+        <link rel="stylesheet" href="@routes.Assets.versioned("lib/web/css/bootstrap-datetimepicker.min.css")">
```

Comment:
would be better if it's managed by webjars https://www.webjars.org/ with npm dependency (see build.sbt:60)